<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>主页</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;华文中宋&quot;;
color: rgb(85, 85, 255);</string>
      </property>
      <property name="text">
       <string>--刷新--</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>400</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;华文中宋&quot;;
color: rgb(85, 85, 255);</string>
      </property>
      <property name="text">
       <string>下一页</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>400</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;华文中宋&quot;;
color: rgb(85, 85, 255);</string>
      </property>
      <property name="text">
       <string>上一页</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>400</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 85, 255);
font: 10pt &quot;华文中宋&quot;;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QTableView" name="QuestionList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>621</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_sort">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>10</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>搜索</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_keyword">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_search">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>10</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_result">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>531</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_search_way">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>我的</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;华文中宋&quot;;
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>退出登录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;华文中宋&quot;;</string>
      </property>
      <property name="text">
       <string>我的问题</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;华文中宋&quot;;</string>
      </property>
      <property name="text">
       <string>我关注的</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;华文中宋&quot;;</string>
      </property>
      <property name="text">
       <string>我要提问</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;华文中宋&quot;;</string>
      </property>
      <property name="text">
       <string>修改密码</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>491</width>
        <height>391</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QTableView" name="myQuestListview">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>521</width>
          <height>361</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="list_up2">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>360</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 85, 255);
font: 12pt &quot;华文中宋&quot;;</string>
        </property>
        <property name="text">
         <string>上一页</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>360</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 85, 255);
font: 12pt &quot;华文中宋&quot;;</string>
        </property>
        <property name="text">
         <string>1/1</string>
        </property>
       </widget>
       <widget class="QPushButton" name="list_down2">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>360</y>
          <width>51</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 85, 255);
font: 12pt &quot;华文中宋&quot;;</string>
        </property>
        <property name="text">
         <string>下一页</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QTableView" name="focuslistView">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>491</width>
          <height>361</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="list_up3">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>360</y>
          <width>51</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 85, 255);
font: 12pt &quot;华文中宋&quot;;</string>
        </property>
        <property name="text">
         <string>上一页</string>
        </property>
       </widget>
       <widget class="QPushButton" name="list_down3">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>360</y>
          <width>51</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;华文中宋&quot;;
color: rgb(85, 85, 255);</string>
        </property>
        <property name="text">
         <string>下一页</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>360</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 85, 255);
font: 12pt &quot;华文中宋&quot;;</string>
        </property>
        <property name="text">
         <string>1/1</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_hostname">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 85, 255);
font: 14pt &quot;华文中宋&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>当前用户：</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>645</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
